File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/rtriangle.py
from Quartz import CGPathCreateWithEllipseInRect,CGPathCreateMutable,CGPathMoveToPoint, CGPathCloseSubpath, CGPathAddLineToPoint

name = "Right Triangle"

def create_cgpath(frame):
    newPath = CGPathCreateMutable()
    CGPathMoveToPoint(newPath, None, 0,0)
    CGPathAddLineToPoint(newPath, None, frame.size.width, 0)
    CGPathAddLineToPoint(newPath, None, 0, frame.size.height)
    CGPathCloseSubpath(newPath)
    return newPath

File: CapturePlugins/CSShapeCapturePlugin/CSShapeCapturePlugin/ShapePaths/rtriangle.js
var name = "Right Triangle"

var createPath = function(newPath, frame)
{
    newPath.moveToPoint({x:0, y:0});
    newPath.lineToPoint({x:frame.width, y:0});
    newPath.lineToPoint({x:0, y:frame.height});
    newPath.closePath();
}
